ui/main/src/app/business/view/userCard/userCard.view.ts

```diff
@@ -185,6 +194,9 @@ export class UserCardView {
     public userSelectsSeverity(severity: Severity) {
         this.severityForm.userSelectsSeverity(severity);
     }
+    public userSelectsKeepChildCards(keepChildCards: boolean) {
+        this.keepChildCardsForm.userSelectsKeepChildCards(keepChildCards);
```

Comment:
If i  comment the line test are passing successfully , it shoud not .

Add  a test in  ui/main/src/app/business/view/userCard/userCard.view.spec.ts  (in   ` describe('Get ChildCard for preview ', async () => {` )